jekyll/_cci2/configuration-reference.md

```diff
@@ -1123,8 +1119,8 @@ version: 2.1
 jobs:
   build:
     machine:
-      image: ubuntu-1604-cuda-10.1:201909-23
-    resource_class: gpu.nvidia.small
+      image: linux-cuda-12:default
```

Comment:
Are these tags supposed to be default?

Reply:
yes, these are intended to be default per the [image policy](https://circleci.com/docs/linux-cuda-images-support-policy/)